src/components/Header/Header.css

```diff
@@ -0,0 +1,21 @@
+
+.btn-login,.hamburger {
+  /*  transparent by default */
+  background: white; 
+}
+.btn-login:hover,.hamburger:hover {
+  /* primary color by default */
+  border-color: white; 
+}
+
+.input {
+  background: white;
+}
```

Comment:
can add input class with 2nd line because same properties
also remove comments if not necessary

Reply:
Sure will do it in next PR!